renderer/lib/cast.js

```diff
@@ -37,7 +38,7 @@ function chromecastPlayer (player) {
     var torrentSummary = state.saved.torrents.find((x) => x.infoHash === state.playing.infoHash)
     player.play(state.server.networkURL, {
       type: 'video/mp4',
-      title: config.APP_NAME + ' — ' + torrentSummary.name
+      title: config.APP_NAME + ' - ' + torrentSummary.name
```

Comment:
its funny, for some weird reason the long dash breaks the DLNA service, i think I copied it back here to create consistency with the other call (maybe reuse it in the future?)